src/meltano/cli/config.py

```diff
@@ -131,6 +131,7 @@ def get_label(metadata) -> str:
     default="json",
 )
 @click.option("--extras", is_flag=True, help="View or list only plugin extras.")
+@click.option("--unsafe", is_flag=True, help="Expose values for sensitive settings.")
```

Comment:
Wdyt about this
```suggestion
@click.option(
    "--safe/--unsafe",
    default=True,
    help="Expose values for sensitive settings.",
)
```

Then you can change instances of `not unsafe` to safe.

Reply:
100% agree - changed in d4b7e59.